### PR TITLE
Add vendors and styling for 404 page

### PR DESCRIPTION
**Pull request by kaankabalak**

### CODE REVIEW

**Comment by deekoder** [comment]
We are adding this css as a placeholder to 404.erb in the future.

File: lib/scss/inc/_mixins.scss [hunk 1]
```diff
@@ -1,3 +1,20 @@
+// Transform - Rotate
+@mixin rotate($degrees) {
```
Comment (rushenn) [inline lib/scss/inc/_mixins.scss]: We do not need mixins for vendor prefixes as we are using PostCSS.

File: lib/scss/inc/_mixins.scss [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+  -webkit-transform: rotate($degrees);
+  -ms-transform: rotate($degrees);
+  -o-transform: rotate($degrees);
+  transform: rotate($degrees);
+}
+
+// Transition
+@mixin transition($transition, $transition-duration) {
```
Comment (rushenn) [inline lib/scss/inc/_mixins.scss]: Same here...

**Comment by rushenn** [comment]
@kaankabalak @deekoder We are using rather small number of icons throughout the project and for which we do not require any icon frameworks such as FontAwesome. I've used SVG icons in the header and we can follow the same way for any other future iconography in order to reduce the overall project package size.

**Comment by kaankabalak** [comment]
Thanks for the constructive feedback @rushenn, will implement the changes shortly

**Comment by kaankabalak** [comment]
@rushenn, updated the PR, would appreciate if you could take another look

**Pull request merged**